settings.gradle.kts

```diff
@@ -6,7 +6,6 @@ include(":here-naksha-lib-psql")
 //include(":here-naksha-lib-extension")
 include(":here-naksha-handler-activitylog")
 //include(":here-naksha-handler-http")
-//include(":here-naksha-handler-psql")
```

Comment:
if we're sure to completely remove everything related to the psql handler, it's safe to delete this completely instead of commenting out. The same can be said to the other modules here.

Reply:
"handler-psql" is deleted (not commented) :) 
I don't intend to remove other modules for now, as not 100% sure whether it is OK to lose them.

Reply:
Ah good, sorry I saw it wrong :)